### DISCUSSION TITLE
在使用图片旋转功能时，会导致多张图片被旋转。并且由win11自带旋转的图片，再使用自动标注后，方向出现的问题

### DISCUSSION THREAD

**Discussion by TTTalen** [PFCCLab/PPOCRLabel#16] [Q&A] [answered]
Q1:旋转功能，由于会导致多找图片旋转无法使用。
Q2:如下图所示，标签中的内容，能明确知道，标识的为此图中的有效信息，但是位置错误，若使用Ctrl+Shift+R可正确识别内容，但是位置错误导致的，自动表示仍不可用
<img width="854" alt="屏幕截图 2024-06-14 181728" src="https://github.com/PFCCLab/PPOCRLabel/assets/103922971/2bc69fe2-97b5-4c60-b208-703049be3367">

**Comment by GreatV** [comment]
这可能和图片中的方向信息有关系，估计是代码里没有匹配上。

**Reply by TTTalen** [reply, marked as answer]
https://github.com/PFCCLab/PPOCRLabel/blob/94cc9de1b5c3fc6731a96b46155c45f07b47e3b2/PPOCRLabel.py#L1270
def rotateImgAction(self, k=1, _value=False):
    filename = self.mImgList[self.currIndex]
这里是获取当前图片路径后进行旋转操作吧，如果我的训练集图片使用删除图像功能后，会导致坐标信息保存在Label.txt标签里，是否会影响读取到的filename 。
在这里，如果直接使用filename = self.filePath，会不会好一点

**Reply by GreatV** [reply]
@TTTalen  有道理，你可以提一个PR，修一下。
